### ISSUE TITLE
banaction_allports not part of config options?

### ISSUE THREAD

**Issue by jeanmonet** [Oefenweb/ansible-fail2ban#62]
Hi - first of all thank you for your work on this role.

Regarding possible configuration options, is there a reason for which `banaction_allports` variable is not proposed? Or can it be added?

**Comment by tersmitten** [comment]
I'm not sure that I know what the purpose of it will be. Can you point me to some documentation or example?

**Comment by jeanmonet** [comment]
I'm not an expert on fail2ban, but I was assuming `banaction_allports` can be used, for example, in the following way:

```ini
# jail.local
[sshd]
banaction: "%(banaction_allports)s"
...
```

The goal in this example would be to have each ban issued by the `sshd` jail be applicable to all the server's ports.
Let me know if I'm mistaken.

**Comment by tersmitten** [comment]
I think that this should work:

```yaml
---
- hosts: all
  roles:
    - fail2ban
  vars:
    fail2ban_services:
      - name: sshd
        banaction: '"%(banaction_allports)s"'
```

**Comment by jeanmonet** [comment]
Won't this apply `banaction_allports` default, which is `iptables-allports`? If one wants `banaction_allports` to use `ufw` instead, one can manually set `banaction_allports = ufw` (will test again behavior a bit later today, but that's my understanding). The role doesn't currently provide the ability to set `fail2ban_banaction_allports: ufw` (for example).

**Comment by tersmitten** [comment]
Fixed in #64?

Feel free to reopen.
